README.md

```diff
@@ -7,11 +7,12 @@ The service is backed by RabbitMQ. All outgoing Iroha blocks are stored in the R
 Chain-adapter functionality may be used by utilizing the `com.d3.chainadapter.client.ReliableIrohaChainListener` class. The class and the service itself are written in Kotlin programming language that has great interoperability with Java.
 
 The service is fail-fast, i.e it dies whenever Iroha or RabbitMQ goes offline.
-## Configuration file overview
-Chain adapter uses `chain-adapter.properties` as a default configuration file that is located in `resources` folder inside the project. However, every configuration item could be changed through environmental variables. 
-
+## Server configuration file overview
+Chain adapter uses `chain-adapter.properties` as a default configuration file that is located in the `resources` folder inside the project. However, every configuration item could be changed through environmental variables.
```

Comment:
"inside the chain-adapter project (not in the project where chain-adapter is used)." - it will be good to mention it (IMHO)